### DISCUSSION TITLE
Two factor authentication

### DISCUSSION THREAD

**Discussion by aliakbaresmaeiliiii** [community/community#136426] [Other Feature Feedback, Questions, & Ideas] [answered]
Urgent: Account Recovery Assistance Needed

General

### GitHub Feature Area

Mobile

### Body

Dear GitHub Support Team,

I hope this message finds you well. My name is Aliakbar Esmaeili, and I am reaching out to seek your urgent assistance in recovering access to my GitHub account associated with the email address aliakbaresmaeili98@gmail.com.

Recently, I had to reset my phone, which unfortunately resulted in the deletion of my GitHub authentication app. As a result, I am currently unable to log in to my GitHub account or access any of my projects. These projects are crucial to my ongoing work, and I am unable to proceed without regaining access to my account.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @aliakbaresmaeiliiii ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). . The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place once submitted. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note:  The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._

**Reply by appletorsh19-stack** [reply]
سلام اکانت من و خانواده ام حک شده است میخواهم کمکم کنید کلیدهای جانبی بسته شود و برنامه های در حال اجرا متوقف شود

**Reply by sallyhatm** [reply]
Hello,

I need to recover my photos and videos linked to the account registered with this email. The app is installed on my other phone, but I forgot the phone's lock code and can't access it. I still have the email and password for the account, so I kindly request your assistance in restoring my data or providing a way to access the account from another device. Thank you for your help. 
sallyhatm1103@gmail.com

**Comment by sbahhj** [comment]
اقترح ان يكون اتصالا هاتفيا والا يزيد مشقة على المستخدم

**Comment by appletorsh19-stack** [comment]
#3889 
#39368 
#178778 
#149939
